test/requests/support/rubocop_runner_test.rb

```diff
@@ -0,0 +1,26 @@
+# typed: true
+# frozen_string_literal: true
+
+require "test_helper"
+require "ruby_lsp/requests/support/rubocop_runner"
+
+module RubyLsp
+  module Requests
+    module Support
+      class RuboCopRunnerTest < Minitest::Test
+        def test_adds_raise_cop_error_flag_if_rubocop_version_supports_it
+          runner = RuboCopRunner.new("--lint")
+
+          assert(runner.instance_variable_get(:@options).fetch(:raise_cop_error))
+        end
+
+        def test_does_not_add_raise_cop_error_flag_if_rubocop_version_does_not_support_it
+          skip("fix this")
+          runner = RuboCopRunner.new("--lint")
+
+          refute(runner.instance_variable_get(:@options).key?(:raise_cop_error))
```

Comment:
Actually, I think this assertion is assuming more about the internals of our code than stubbing RuboCop internals.